policies/linux.consul.-.catalog.nodes_warning.json

```diff
@@ -0,0 +1,28 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 2,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 0,
+        "metric": "consul.catalog.nodes_warning",
+        "operator": ">"
+      }
+    ],
+    "deleted": false,
+    "description": "consul.catalog.nodes_warning is greater than 0",
```

Comment:
Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken?

Reply:
I've commited the requested changes to the branch